libwnbd/wnbd_ioctl.cpp

```diff
@@ -95,10 +99,17 @@ DWORD WnbdOpenAdapterEx(PHANDLE Handle, PDEVINST CMDeviceInstance)
         *CMDeviceInstance = DevInfoData.DevInst;
     }
     else {
-        LogError(
-            "Could not oped WNBD device. Please make sure that the "
-            "driver is installed. Error: %d. Error message: %s",
-            ErrorCode, win32_strerror(ErrorCode).c_str());
+        if (ErrorCode == ERROR_ACCESS_DENIED) {
```

Comment:
`win32_strerror` translates the error code using `FormatMessage` which should provide a meaningful error though.

I understand the need to treat this error differently but I'm not really in favor of it.

Reply:
FormatMessage will just return "Access denied" without any other details. Using unelevated prompts is quite common and most people wouldn't figure out right away the cause of the "Access denied" error, so I really think we should add a more explicit message. There's no downside to it and we'll spare people from some wasted time debugging this, maybe even avoid some bug reports.